### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 6

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#501] [Esercizi M-Z]
![es.6](https://github.com/user-attachments/assets/4073ed79-5eb1-4dd8-81be-2cd084b6e868)

**Comment by Vinz002** [comment]
![Immagine WhatsApp 2024-10-19 ore 20 32 15_ed3baa6f](https://github.com/user-attachments/assets/4dcc67c8-ab92-421d-8f8e-1da2aaea7d89)
